docs/concepts.rst

```diff
@@ -1017,6 +1017,32 @@ This animated gif shows the UI interactions. TaskGroups are expanded or collapse
 
 .. image:: img/task_group.gif
 
+@taskgroup decorator
+--------------------
+
+TaskGroup can be created using @taskgroup decorator, it takes one mandatory argument 'group_id' which is same as constructor of TaskGroup class. It works exactly same as creating TaskGroup using context manager 'with TaskGroup('groupid') as section:'.
+
+.. code-block:: python
+
+  @task
+  def task_1(value):
+      return f'[ Task1 {value} ]'
+
+
+  @task
+  def task_2(value):
+      print( f'[ Task2 {value} ]')
+
+
+  @taskgroup(group_id='section_1')
+  def section_1(value):
+      return task_2(task_1(value))
+
```

Comment:
I'm not sure if we need two examples. If we decide to keep both the it would be nice to add some text between them.